test/fuzz/luaL_loadbuffer/serializer.h

```diff
@@ -20,6 +20,9 @@ using namespace lua_grammar;
 /**
  * Fuzzing parameters:
  * kMaxCounterValue - value in `if counter > kMaxCounterValue then break end`.
+ * kMaxDepth - maximum depth of code to be generated.
+ * Time of execution will not exceed O(kMaxCounterValue ^ kMaxDepth * N) where
```

Comment:
whitespace after 0

Reply:
This is big O notation, in tarantool used without space: https://github.com/tarantool/tarantool/blob/ef6335e862ac9110e4bf1d146a839cb9c35b7ecd/src/box/key_def.h#L566C22-L566C26.

Also, this part was dropped.